src/gallium/drivers/lima/lima_draw.c

```diff
@@ -556,6 +557,57 @@ lima_stencil_op(enum pipe_stencil_op pipe)
    return -1;
 }
 
+static int
+lima_compare_func(enum pipe_compare_func func)
```

Comment:
Seems the enums are already what the value we need, so we don't need to convert them explicitly.

Reply:
I think it's better to keep it this way just in case if enum changes. Relying on enum to be some specific value is fragile.

Reply:
This enum order should be like this in purpose and pretty stable. Mali HW also seems code some value align with OpenGL spec like the pipe_prim_type, so I just use it in lima_pack_plbu_cmd.

Reply:
OK, fixed.